### DISCUSSION TITLE
Test a group of tests only with Japa

### DISCUSSION THREAD

**Discussion by LeCoupa** [adonisjs/core#3741] [Help] [answered]
Is there a way to only test a specific suite of tests?

`test.only.group` or `test.group.only` does not seem to work.

<img width="451" alt="CleanShot 2022-05-27 at 12 29 18@2x" src="https://user-images.githubusercontent.com/1658644/170682943-22504314-b9b8-456b-a437-c05f639e840e.png">

**Comment by Kuzzy** [comment, marked as answer]
You can use `pin` method: https://japa.dev/filtering-tests#pinning-tests
